uninstall.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+#
+# uninstall/removes g910-gkeys.
+#
+# options:
+#  -a: will also remove /etc/g910-gkeys/ configuration directory.
+#  -d: dry-run: will display actions without doing anything
+#
+# TODO: remove possible faulty previous versions files (in /etc/systemd/system
```

Comment:
Uninstaller should only remove files that were created on this particular install.

Reply:
directories were created during install...

Reply:
The comment was regarding todo: removing possible faulty previous versions, not folders that don't get deleted on uninstall.